datalogging/can_recieve.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/brutella/can"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("Usage: go run can_reciever.go <INTERFACE NAME>")
+		return
+	}
+	var interfce_name = os.Args[1]
+
+	intrfce, err := net.InterfaceByName(interfce_name)
+
+	if err != nil {
+		fmt.Printf("Could not connect to %s. Check that you have added it properly.\n", interfce_name)
+		os.Exit(1)
+	}
+
+	// Establish connection to CAN bus
+	connection, _ := can.NewReadWriteCloserForInterface(intrfce)
+	can_bus := can.NewBus(connection)
```

Comment:
we should be consistent in naming our variables. The [Go style guide](https://google.github.io/styleguide/go/decisions#naming) suggests using `camelCase` instead of `snake_case`